lib/platform/github/index.ts

```diff
@@ -113,10 +115,20 @@ export async function initPlatform({
       discoveredGitAuthor = `${userDetails.name} <${userEmail}>`;
     }
   }
+  const gitIgnoredAuthors = [];
+  if (platformCommit) {
+    userDetails = await getUserDetails(defaults.endpoint, token);
+    const userId = userDetails.id.toString();
+    const userName = userDetails.username;
+    const userEmail = `${userId}+${userName}@users.noreply.github.com`;
+    const platformAuthor = `${userDetails.username} <${userEmail}>`;
+    gitIgnoredAuthors.push(platformAuthor);
+  }
```

Comment:
This throws when using an app token, it don't have access to user api.

Reply:
You're referring to something other than these scopes for my personal user token?

![image](https://user-images.githubusercontent.com/1239644/139805250-2bc57a0a-44ab-4c28-a350-0b9af8bad8ad.png)

Because it works good for me even without user scopes 🤔

Reply:
He means the github app token, e.g. WhiteSource Renovate. To avoid that API call, we specify `gitAuthor` and `username` manually.

Reply:
Just reverted automatic ignored authors auto-detection, so now `gitIgnoredAuthors` option should be set manually